src/sentry/conf/server.py

```diff
@@ -655,6 +656,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("unmerge", routing_key="unmerge"),
     Queue("update", routing_key="update"),
     Queue("profiles.process", routing_key="profiles.process"),
+    Queue("release_health.duplex", routing_key="release_health.duplex"),
```

Comment:
@beezz Will this queue be auto-created in production, or is there an extra step necessary for this?

Reply:
It will be auto-created we just need to make sure it's federated, adding it here ~ https://github.com/getsentry/ops/blob/master/cookbooks/getsentry/attributes/default.rb#L336-L375

Reply:
@beezz thanks, should I open a PR there as well? Does that need to be merged/deployed before this PR?